app/pages/learn/01_tutorial/03_getting-to-know-the-language/02_basics/03_creating-arrays.md

```diff
@@ -209,6 +209,64 @@ Mr. Smith
 Ms. Jones
 ```
 
+<a id="jagged">&nbsp;</a>
```

Comment:
I think this would need to be added to the table of contents (see the beginning of the Markdown file).
At that point, you might also want to change the last updated timestamp.